code/modules/projectiles/updated_projectiles/guns/specialist.dm

```diff
@@ -470,7 +470,7 @@
 	throw_speed = 2
 	throw_range = 10
 	force = 5.0
-	wield_delay = 8
+	wield_delay = 6
```

Comment:
```suggestion
	wield_delay = 0.6 SECONDS
```

Reply:
i'm gonna test the commit suggestion button and i'll probably end up regretting it but curiosity wins